app/Services/StatementSearchService.php

```diff
@@ -390,6 +390,25 @@ public function bulkIndexStatements(Collection $statements): void
         }
     }
 
+    public function bulkDeleteStatements(\Illuminate\Support\Collection $statements): void
```

Comment:
maybe move to another service than the Search service ?

Reply:
This is for removing statements from the search index. at present I am not doing any delete so yes i can moved to separate it better.